Source/GSStoryboardTransform.m

```diff
@@ -0,0 +1,828 @@
+      NSXMLNode *attr = [ce attributeForName: @"id"];
+      controllerId = [attr stringValue];
+      
+      FOR_IN(NSXMLElement*, o, windowControllers)
+        {
+          NSXMLElement *objects = (NSXMLElement *)[o parent];
+          NSArray *windows = [o nodesForXPath: @"//window" error: NULL];
+          FOR_IN(NSXMLNode*, w, windows)
+            {
+              [w detach];
+              [objects addChild: w];
+            }
+          END_FOR_IN(windows);
+        }
+      END_FOR_IN(windowControllers);
+    }
+  
+  if ([viewControllers count] > 0)
+    {
+      NSXMLElement *ce = [viewControllers objectAtIndex: 0];
+      NSXMLNode *attr = [ce attributeForName: @"id"];
+      controllerId = [attr stringValue];
+    }
+  
+  if ([controllerPlaceholders count] > 0)
+    {
+      NSXMLElement *ce = [controllerPlaceholders objectAtIndex: 0];
+      NSXMLNode *attr = [ce attributeForName: @"id"];
+      controllerId = [attr stringValue];
+    }
+  
+  return controllerId;
+}
+
+- (void) processCustomElement: (NSXMLElement *)coel
+                  forDocument: (NSXMLDocument *)document
+{
+  NSXMLNode *attr = [coel attributeForName: @"sceneMemberID"];
+  if ([[attr stringValue] isEqualToString: @"firstResponder"])
+    {
+      NSXMLNode *customClassAttr = [coel attributeForName: @"customClass"];
+      NSXMLNode *idAttr = [coel attributeForName: @"id"];
+      NSString *originalId = [idAttr stringValue];
+      
+      [idAttr setStringValue: @"-1"]; // set to first responder id
+      [customClassAttr setStringValue: @"FirstResponder"];
+      
+      // Actions
+      NSArray *cons = [document nodesForXPath: @"//action" error: NULL];
+      FOR_IN(NSXMLElement*, celem, cons) 
+        {
+          NSXMLNode *targetAttr = [celem attributeForName: @"target"];
+          NSString *val = [targetAttr stringValue];
+          if ([val isEqualToString: originalId])
+            {
+              [targetAttr setStringValue: @"-1"];
+            }
+        }
+      END_FOR_IN(cons);
+      
+      // Outlets
+      cons = [document nodesForXPath: @"//outlet" error: NULL];
+      FOR_IN(NSXMLElement*, celem, cons)
+        {
+          NSXMLNode *attr = [celem attributeForName: @"destination"];
+          NSString *val = [attr stringValue];
+          if ([val isEqualToString: originalId])
+            {
+              [attr setStringValue: @"-1"];
+            }
+        }
+      END_FOR_IN(cons);
+    }
+}
+
+- (void) processStoryboard: (NSXMLDocument *)xml
+{
+  NSArray *docNodes = [xml nodesForXPath: @"document" error: NULL];
+
+  if ([docNodes count] > 0)
+    {
+      NSXMLElement *docNode = [docNodes objectAtIndex: 0];
+      NSArray *array = [docNode nodesForXPath: @"//scene" error: NULL];
+      
+      // Set initial view controller...
+      ASSIGN(_initialViewControllerId, [[docNode attributeForName: @"initialViewController"] stringValue]);             
+      FOR_IN(NSXMLElement*, e, array) 
+        {
```

Comment:
Just a reminder as you probably already know this: You don't need the "{” after a FOR_IN statement. There is already one automatically added. It doesn't do any harm, you may always add braces in your code as you like.